lib/contracts/language-variant-contracts.ts

```diff
@@ -37,6 +42,12 @@ export namespace LanguageVariantContracts {
         pagination: SharedContracts.IPaginationModelContract;
     }
 
+    export interface IUpsertLanguageVariantWorkflowContract {
+        workflow_identifier: SharedContracts.IReferenceObjectContract;
```

Comment:
Both `workflow_identifier` and `step_identifier` do not support `external_id` property which is included within `SharedContracts.IReferenceObjectContract`. I guess you should make a new shared interface that contains only codename & id (both optional) properties and use it here.